### PR TITLE
docs: development: document how to run L1 invalidation and challenge checks

### PR DESCRIPTION
**Pull request by istankovic**
Closes: #2121

### CODE REVIEW

**Review by bilbeyt — APPROVED** [review]

File: docs/source/development.rst [hunk 1]
```diff
@@ -271,3 +271,118 @@ Also, agent releases are numbered in line with the deployment branch name.
...
+
+.. note:: The test token `must` be an ERC20 token.
+
+   In general, it's easiest to just deploy ``MintableToken`` when deploying Beamer
+   (see ``--deploy-mintable-token`` option of the ``beamer deploy`` command).
+
+   From this point on, we will assume that ``MintableToken`` (``TST``) will be used.
+
+#. Configure and start a single agent instance that monitors all ``X1`` ... ``Xn`` chains.
+   The test token address needs to be configured properly on each chain in the agent
```
Comment (fredo) [inline docs/source/development.rst:344]: There is the information missing that the token also needs to be configured in the contracts (transfer limit > 0 and ideally ethInToken = 0)

File: docs/source/development.rst [hunk 2]
```diff
@@ -271,3 +271,118 @@ Also, agent releases are numbered in line with the deployment branch name.
...
+   configuration file and the agent needs to be whitelisted on all chains.
+
+.. note:: Make sure that the agent version you are going to use ships with a relayer
+   that is aware of all the chains, including the one you just deployed to.
+
+#. Fund the agent on each chain. This means that, on each chain, the agent needs to
+   have at least 1 wei of the test token, as well as at least 0.0015 ETH for staking
```
[the remark this answers is not in the document]
Comment (istankovic) [reply docs/source/development.rst:351]: Done.

File: docs/source/development.rst (hunk 1 above)
Comment (istankovic) [reply docs/source/development.rst:348]: Good point, added a separate step describing that. :+1:

**Review by fredo — APPROVED** [review]
Looks good

**Pull request merged**